gcc/rust/ast/rust-ast-full-test.cc

```diff
@@ -4010,7 +4010,15 @@ filename_from_path_attribute (std::vector<Attribute> &outer_attrs)
       return "";
     }
 
-  auto path = path_attr.get_attr_input ().as_string ();
+  auto path_value = path_attr.get_attr_input ().as_string ();
+
+  // At this point, the 'path' is of the following format: '= "<file.rs>"'
+  // We need to remove the equal sign and only keep the actual filename.
+  // In order to do this, we can simply go through the string until we find
+  // a character that is not an equal sign or whitespace
+  auto filename_begin = path_value.find_first_not_of ("=\t ");
```

Comment:
I wonder is it worth checking in Rustc to see if their Attributes structure cleans this up for you in the parser so you get a structure like:

string attr_name, bool has_rhs; string get_rhs;

Reply:
I think we will need some type of code like that for attributes in general so lets make an issue. Don't change it for this PR.

Reply:
https://github.com/Rust-GCC/gccrs/issues/665